zerver/openapi/openapi.py

```diff
@@ -117,28 +188,51 @@ def get_openapi_return_values(endpoint: str, method: str,
     response = response['properties']
     return response
 
-exclusion_list: List[str] = []
-
 def validate_against_openapi_schema(content: Dict[str, Any], endpoint: str,
                                     method: str, response: str) -> None:
     """Compare a "content" dict with the defined schema for a specific method
     in an endpoint.
     """
+    if endpoint in EXCLUDE_ENDPOINTS:
+        return
+    # Collision between /users/me/subscriptions/{id} and /users/{id}/subscripitions/{id}
+    if endpoint.startswith('/users/me/subscriptions/'):
+        return
```

Comment:
We should have this differently; those are 2 different endpoints that should each get their own validation.  One should be able to tell which is which based on the `"me"` vs. an integer in there.

Reply:
The new algo has no collisions so this part is now unnecessary and has been removed.